web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -570,7 +571,7 @@ input.button {
   will-change: background-color;
 }
 .form-actions input.button {
-  margin-top: 8px;
+  margin-top: 50px;
```

Comment:
If you're trying to push the whole action sections down, it might be better to put the top margin on .form-actions (or maybe as a padding if its to do with border placement)